site/getting-started.md

```diff
@@ -17,15 +17,39 @@ Before you start you will need:
 
 ## Add Contour to your cluster
 
-Run:
+### Option 1
+
+Deploy Contour using the rendered manifest:
 
 ```bash
 $ kubectl apply -f {{ site.url }}/quickstart/contour.yaml
 ```
 
+### Option 2
+
```

Comment:
Also note that we have guidance for using Helm incoming as well in #3125, thanks to @georgegoh.